test/integration/services/price.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { PriceService } from "@services/price"
```

Comment:
could this be a unit test?

Reply:
if we mock db yes, but in this case is to validate if the query to mongodb is returning the correct interval